python/tvm/relay/op/nn/_nn.py

```diff
@@ -141,7 +142,17 @@ def convert_conv2d(attrs, inputs, tinfos, desired_layout):
     from tvm import relay
     data, weight = inputs
     new_attrs = dict(attrs)
+    desired_layout = str(desired_layouts[0])
```

Comment:
Since we now have `desired_kernel_layout`, would that be better to rename this one to `desired_data_layout` (so do the other conv2ds)?